tbxforms/forms.py

```diff
@@ -11,7 +11,7 @@
     from wagtail.contrib.forms.forms import FormBuilder
 
 
-class BaseForm:
+class TbxFormsMixin:
```

Comment:
will need to be highlighted in upgrade considerations

Reply:
Good shout, added: https://github.com/torchbox/tbxforms/releases